src/components/CameraPermissions/Recover/index.tsx

```diff
@@ -38,7 +38,10 @@ const Recover: FunctionComponent<Props> = ({ translate, audio }) => {
     <Button
       variant="primary"
       className={classNames(theme['button-centered'], theme['button-lg'])}
-      onClick={() => window.location.reload()}
+      onClick={trackClick(
+        () => window.location.reload(),
+        'screen_face_selfie_capture_camera_access_denied_refresh_button'
```

Comment:
This will go for all screens that use it, not only face selfie. Can you replace it with a generic name?